keras_cv/ops/point_cloud.py

```diff
@@ -12,13 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import numpy as np
 import tensorflow as tf
 
 
 def get_rank(tensor):
     return tensor.shape.ndims or tf.rank(tensor)
 
 
```

Comment:
nit:

lets abide to Keras naming here and avoid acronyms; i.e. use `radians` instead of `rads`.

Reply:
+1

Reply:
Done.